examples/url-shortener.php

```diff
@@ -60,7 +60,13 @@
  * local access token in this case
  ************************************************/
 if (isset($_REQUEST['logout'])) {
-  unset($_SESSION['access_token']);
+  if (validateCsrfToken()) {
```

Comment:
is any of this necessary? I don't understand why we need to remove the csrf token on logout, or why the validate function needs to be called. At the very least we can just unset the token.